task1/Program.cs

```diff
@@ -45,5 +71,792 @@ public static decimal PerformCalculation(int r)
+            int counter = 0;
+            int temp = testNumber;
+            for (int i = 0; i < inputNumber.Length; i++)
+            {
+                if ((temp - temp / 10 * 10) % 2 == 0)
+                {
+                    counter++;
+                }
+                temp = temp / 10;
+            }
+            Console.WriteLine($"{counter} even cyfres in {testNumber}");
+        }
+
+        public static void GetArithmeticalMean()
+        {
+            Console.WriteLine("Input lower number");
+            int lowerNumber = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Input higher number");
+            int higherNumber = Convert.ToInt32(Console.ReadLine());
+            int sum = 0;
+            int count = 0;
+            for (int i = lowerNumber; i <= higherNumber; i++)
+            {
+                sum += i;
+                count++;
+            }
+            double mean = (double
+)sum / count;
+            Console.WriteLine($"Arithmetical mean of numbers from {lowerNumber} to {higherNumber} is {mean}");
+        }
+
+        public static void Skier()
+        {
+            double way = 10.0;
+            int days = 0;
+            for (int i = 2; i > 1; i++)
+            {
+                way = way + way * 1.1;
+                if (way >= 100)
+                {
+                    days = i;
+                    break
+;
+                }
+            }
+            Console.WriteLine($"Skier will run more than 100 km on the {days} day");
+        }
+
+        public static void MultiplyTwoNumbers()
+        {
+            Console.WriteLine("Input first number");
+            int firstNumber = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Input second number");
+            int secondNumber = Convert.ToInt32(Console.ReadLine());
+            int multiplication = 0;
+            if ((firstNumber > 0 && secondNumber > 0) || (firstNumber < 0 && secondNumber < 0))
+            {
+                multiplication = Multiply(Math.Abs(firstNumber), Math.Abs(secondNumber));
+            }
+            if ((firstNumber > 0 && secondNumber < 0) || (firstNumber < 0 && secondNumber > 0))
+            {
+                multiplication = -Multiply(Math.Abs(firstNumber), Math.Abs(secondNumber));
+            }
+            Console.WriteLine($"{firstNumber} * {secondNumber} = {multiplication}");
+
+        }
+
+        private static int Multiply(int firstNumber, int secondNumber)
+        {
+            int result = 0;
+            for (int i = 1; i <= firstNumber; i++)
+            {
+                result += secondNumber;
+            }
+            return result;
+        }
+
+        public static void SquaresBeforeGivenNumber()
+        {
+            Console.WriteLine("Input number");
+            int number = Convert.ToInt32(Console.ReadLine());
+            for (int i = 1; i >= 1; i++)
+            {
+                if (i * i <= number)
+                {
```

Comment:
Since you have a single-line else block and single line if block, no need to place operator brackets.
But it could be used only if it will not affect the readability.